src/Parser/SniffParser.php

```diff
@@ -142,8 +142,9 @@ private function getDocBlock(string $docComment): string
 
         $docBlockSummary = $docBlock->getSummary();
         $docBlockDescription = (string)$docBlock->getDescription();
+        $docBlockDescription = preg_replace('`\n*{@internal[^}]+}`', '', $docBlockDescription);
```

Comment:
* Will line endings always be `\n` at this point ? I.e. have they been normalized ? Or should the regex account for different new line markers `[\n\r]*` ?
* The `{` and `}` characters are regex meta-characters (when used outside of a square bracket character class). Shouldn't these be escaped ?

Also: according to the specs, the inline `{@internal ... }` tag can contain _other_ tags. I'm not saying that needs to be addressed in this PR, however we may want to open an issue to account for this at a later point in time.

> Contrary to other inline tags, the inline version of this tag may contain other inline tags.

Refs: 
- https://docs.phpdoc.org/3.0/guide/references/phpdoc/inline-tags/internal.html#internal
- https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md#55-internal

Reply:
1. We should probably not accept PRs with mixed line endings to PHP code in any case. Mixed line endings should be detected by the CS job on PHPCompatibility (I know for a fact that this sniff exists). I don't mind adding the extra regex, but we should avoid mixed line endings in the first place, as they're trouble for other reasons too.

2. If I escape it, PHPStorm tells me that the escaping character is redundant.

3. I was toying with the idea of leveraging the `getTagsByName` method, but phpDocumentor is unable to detect inline tags. We'd need to either write an extension or smarter regexes, when the time comes.

Reply:
1. That sniff is already in effect for PHPCompatibility, but it is not the job of this project to decide what line endings are acceptable in projects using it. I'm not talking about mixed line endings, but if a project choses to consistently use `\n\r` line endings, the parser should still work correctly.

2. Interesting. I'd still feel it be safer to use it as the PCRE engine has had changes across PHP versions, so it may interpret this differently in different PHP versions.
Also see the specs:
* https://www.php.net/manual/en/regexp.reference.meta.php
* https://www.php.net/manual/en/regexp.reference.escape.php

And per the specs:
> it is always safe to precede a non-alphanumeric with "\" to specify that it stands for itself

3. A smarter regex will probably be the way to go. Might be worth it to open an issue in phpDocumentor about it if needs be. I don't know the exact current status, but a while back it was a known fact that not all documentation features were fully supported yet in phpDocumentor 3.x.

Reply:
1. That's a very good point. Done.
2. Making code compatible across versions is the reason I'm contributing to PHPCompatibility :) Done.
3. Created ticket #17 so that we don't forget.